generator/templates/Default/src/plugins/vuetify/icons/application.js

```diff
@@ -0,0 +1,17 @@
+import {
+  faBox,
+} from '<%_ if (options.plugins.includes("fontawesomepro")){ _%> @fortawesome/pro-solid-svg-icons<%_ } else { _%> @fortawesome/free-solid-svg-icons<%_ } _%>';
+
+const icons = [
+ faBox,
+];
+
+/*project specific icons*/
+const identifiers = {
+  product: ['fas', 'box'],
+};
+
+export default {
+  icons,
+  identifiers,
+};
```

Comment:
Deze is ter voorbeeld en dient als voorstel. Je kan deze natuurlijk anders noemen dan application, maar ik denk dat application de juiste lading dekt -- qua naming.